app/scrapers/__init__.py

```diff
@@ -10,25 +8,19 @@
 from baidu import Baidu
 
 scrapers = {
-    'g': Google(),
-    'b': Bing(),
-    'y': Yahoo(),
-    'd': Duckduckgo(),
-    'a': Ask(),
-    'yd': Yandex(),
-    'u': Baidu()
+    'ask': Ask(),
+    'bing': Bing(),
+    'baidu': Baidu(),
+    'duckduckgo': Duckduckgo(),
+    'google': Google(),
+    'yahoo': Yahoo(),
+    'yandex': Yandex()
```

Comment:
Are you sure. Yandex will work fine since #271 exists !
Also after Baidu, 2 search engines support has been added. Please update your branch. 
Also the value of the search engines are returned from `index.html` . Make changes there too.

Reply:
The yandex problem is caused by line https://github.com/fossasia/query-server/blob/master/app/server.py#L55 combined with the fact that 'yahoo'[0] == 'yandex'[0] == y.

We should stop building workarounds for the single letter approach on line 55.